### ISSUE TITLE
Missing newline after progress bar

### ISSUE THREAD

**Issue by roberthstrand** [k8sgpt-ai/k8sgpt#99]
When you run `k8sgpt analyze` and get multiple errors, the first one's number is stuck at the top with the progress bar. The same happens when running with output set to JSON, but with the opening code bracket.

```bash
❯ ./k8sgpt analyze
   0% |                                                                                                                                                                                                                                                       | (0/4, 0 it/hr) [0s:0s]0 
kube-system/kube-prometheus-stack-kube-scheduler(kube-prometheus-stack-kube-scheduler): Service has no endpoints, expected label component=kube-scheduler 

1 kube-system/kube-prometheus-stack-kube-controller-manager(kube-prometheus-stack-kube-controller-manager): Service has no endpoints, expected label component=kube-controller-manager 

2 kube-system/kube-prometheus-stack-kube-etcd(kube-prometheus-stack-kube-etcd): Service has no endpoints, expected label component=etcd 

3 kube-system/kube-prometheus-stack-kube-proxy(kube-prometheus-stack-kube-proxy): Service has no endpoints, expected label k8s-app=kube-proxy
```

```
 ❯ ./k8sgpt analyze -o json
   0% |                                                                                                                                                                                                                                                          | (0/4, 0 it/hr) [0s:0s]{
"kind":"Service","name":"kube-system/kube-prometheus-stack-kube-etcd","error":["Service has no endpoints, expected label component=etcd"],"details":"","parentObject":"kube-prometheus-stack-kube-etcd"}
{"kind":"Service","name":"kube-system/kube-prometheus-stack-kube-proxy","error":["Service has no endpoints, expected label k8s-app=kube-proxy"],"details":"","parentObject":"kube-prometheus-stack-kube-proxy"}
{"kind":"Service","name":"kube-system/kube-prometheus-stack-kube-scheduler","error":["Service has no endpoints, expected label component=kube-scheduler"],"details":"","parentObject":"kube-prometheus-stack-kube-scheduler"}
{"kind":"Service","name":"kube-system/kube-prometheus-stack-kube-controller-manager","error":["Service has no endpoints, expected label component=kube-controller-manager"],"details":"","parentObject":"kube-prometheus-stack-kube-controller-manager"}
```

This is on macOS Ventura 13.2, tried with both fish and bash, and with both regular built in terminal and Hyper terminal.

**Comment by Tanmoy037** [comment]
I would like to work on this, please assign it to me.

**Comment by ArslanYM** [comment]
> @ArslanYM I also tried this but the issue is only for macOS Ventura 13.2, I use ubuntu 22.04, and it's working here. If you're macOS Ventura 13.2 user, give it a try and share you're approach or solution here.

thanks for the heads up.

**Comment by thschue** [comment]
This should be fixed with #217

**Comment by thomasstxyz** [comment]
I also see this behavior (on Mac OS Ventura 13.2.1, M1) on latest commit 9423b53c1dbae3d0762420a0bacbdace9a2c18c9.

```bash
$ ./k8sgpt analyze --explain
   0% |                                                      | (0/4, 0 it/hr) [0s:0s]error getting completion: error, status code: 429, message: You exceeded your current quota, please check your plan and billing details.
Exhausted API quota. Please try again later
```

![image](https://user-images.githubusercontent.com/82174387/230799916-e47a3cc6-fb69-4334-b358-01400a0f166a.png)
